src/models/WalletAddress.js

```diff
@@ -3,13 +3,13 @@ import { satoshiToDecimal } from '../helpers/utility';
 
 export default class WalletAddress {
   @observable address = ''
-  @observable qtum = 0
-  @observable bot = 0
+  @observable naka = 0
+  @observable nbot = 0
 
   constructor(args, convertToDecimal = true) {
```

Comment:
lets remove the `convertToDecimal` param. we should be consistent everywhere and use decimals for any token-related balance. if you see the `models` I also created a convenience satoshi var as well. so here we could have `naka`, `nakaSatoshi`, `bot`, `botSatoshi`.